pages/home/install/local.mdx

```diff
@@ -97,17 +93,21 @@ To run the Engine on it's own, you can run:
 arcade-engine
```

Comment:
I'm curious if readers are just blindly running this command and getting errors because they have already started the engine via `arcade dev`